vocabularies/Common.json

```diff
@@ -867,6 +867,12 @@
             "LocalDataProperty": {
                 "$Type": "Edm.PropertyPath",
                 "@Core.Description": "Path to property that is used to filter the value list with `eq` comparison"
+            },
+            "InitialValueIsSignificant": {
+                "$Type": "Edm.Bool",
+                "$Nullable": true,
```

Comment:
@BerSie nullable Boolean? Better Nullable="false" and DefaultValue="false".

Reply:
added Nullable="false" and changed to DefaultValue="false"